blob/azureblob/example_test.go

```diff
@@ -42,22 +42,33 @@ func Example() {
 	// Creates a *blob.Bucket backed by Azure Storage Account
 	b, _ := azureblob.OpenBucket(ctx, serviceURL, bucketName, &azureOpts)
 
-	// Obtain the native blob attributes
+	// Get the native blob attributes
```

Comment:
Nit: end comments with a period, here and elsewhere.
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Noted & Fixed